### PR TITLE
Add triage for 2023-10-26

### PR DESCRIPTION
**Pull request by rylev**

### CODE REVIEW

File: triage/2023-10-26.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -0.7% | [-0.7%, -0.7%] | 2     |
+| Improvements ✅ <br /> (secondary) | -0.4% | [-0.7%, -0.2%] | 14    |
+| All ❌✅ (primary)                 | -0.7% | [-0.7%, -0.7%] | 2     |
+
+
+On Windows make `read_dir` error on the empty path [#116606](https://github.com/rust-lang/rust/pull/116606) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=62fae2305e5f3a959bd6ad6c20608c118e93648a&end=aec4741d42e10673e7d5fe0f1c96d407801884e8&stat=instructions:u)
```
Comment (Kobzol) [inline triage/2023-10-26.md:122]: This is also noise. We could perhaps say it also for improvements, when we do for regressions :)

File: triage/2023-10-26.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.2%  | [0.1%, 0.2%]   | 5     |
+| Regressions ❌ <br /> (secondary)  | 0.2%  | [0.2%, 0.2%]   | 2     |
+| Improvements ✅ <br /> (primary)   | -0.7% | [-3.6%, -0.2%] | 77    |
+| Improvements ✅ <br /> (secondary) | -1.1% | [-1.8%, -0.4%] | 30    |
+| All ❌✅ (primary)                 | -0.7% | [-3.6%, 0.2%]  | 82    |
+
+
+time: use clock_gettime on macos [#116238](https://github.com/rust-lang/rust/pull/116238) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=f1a5ce19f5aa0cf61ed7b9f75b30e610befeed72&end=e918db897df677d9bbda64d82ebbb8f1bc9e557f&stat=instructions:u)
```
Comment (Kobzol) [inline triage/2023-10-26.md:144]: Same here.

**Review by Kobzol — APPROVED** [review]
Small nit (but I think it's fine to keep it like it is).

File: triage/2023-10-26.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 0.4%  | [0.3%, 0.5%]   | 13    |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -3.7% | [-3.7%, -3.7%] | 1     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | 0.1%  | [-3.7%, 0.5%]  | 14    |
+- Small regression - asked the author and reviewer if they feel like they had a quick idea how to address it.
+
+
+Rollup of 7 pull requests [#117180](https://github.com/rust-lang/rust/pull/117180) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=cf226e93dcb0e21e2daa7c26aca0c5b46ff1b646&end=ab5c841a1f3c09edc5ea07722519627c960aed17&stat=instructions:u)
```
Comment (lqd) [inline triage/2023-10-26.md:215]: What did you think about this one?

The one primary 3.7% regression looks real. The tiny secondary one looks either noise or has basically returned to its prior value. The small improvement also seems to have sustained at the new levels.

File: triage/2023-10-26.md [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+On Windows make `read_dir` error on the empty path [#116606](https://github.com/rust-lang/rust/pull/116606) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=62fae2305e5f3a959bd6ad6c20608c118e93648a&end=aec4741d42e10673e7d5fe0f1c96d407801884e8&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -2.8% | [-2.8%, -2.8%] | 1     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | -2.8% | [-2.8%, -2.8%] | 1     |
+
```
Comment (Mark-Simulacrum) [inline triage/2023-10-26.md:131]:
```suggestion

This is noise.
```

File: triage/2023-10-26.md [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+time: use clock_gettime on macos [#116238](https://github.com/rust-lang/rust/pull/116238) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=f1a5ce19f5aa0cf61ed7b9f75b30e610befeed72&end=e918db897df677d9bbda64d82ebbb8f1bc9e557f&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | -     | -              | 0     |
+| Regressions ❌ <br /> (secondary)  | -     | -              | 0     |
+| Improvements ✅ <br /> (primary)   | -3.6% | [-3.6%, -3.6%] | 1     |
+| Improvements ✅ <br /> (secondary) | -     | -              | 0     |
+| All ❌✅ (primary)                 | -3.6% | [-3.6%, -3.6%] | 1     |
+
```
Comment (Mark-Simulacrum) [inline triage/2023-10-26.md:153]:
```suggestion

This is noise.
```

File: triage/2023-10-26.md [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+Rollup of 7 pull requests [#117180](https://github.com/rust-lang/rust/pull/117180) [(Comparison Link)](https://perf.rust-lang.org/compare.html?start=cf226e93dcb0e21e2daa7c26aca0c5b46ff1b646&end=ab5c841a1f3c09edc5ea07722519627c960aed17&stat=instructions:u)
+
+| (instructions:u)                   | mean  | range          | count |
+|:----------------------------------:|:-----:|:--------------:|:-----:|
+| Regressions ❌ <br /> (primary)    | 3.7%  | [3.7%, 3.7%]   | 1     |
+| Regressions ❌ <br /> (secondary)  | 0.2%  | [0.2%, 0.2%]   | 1     |
+| Improvements ✅ <br /> (primary)   | -     | -              | 0     |
+| Improvements ✅ <br /> (secondary) | -1.3% | [-1.3%, -1.3%] | 1     |
+| All ❌✅ (primary)                 | 3.7%  | [3.7%, 3.7%]   | 1     |
+
```
Comment (Mark-Simulacrum) [inline triage/2023-10-26.md:224]:
```suggestion

The one primary 3.7% regression looks real. The tiny secondary one looks either noise or has basically returned to its prior value. The small improvement also seems to have sustained at the new levels.
```

**Pull request merged**